spec/requests/api/v2/providers/courses/update_gcse_requirements_spec.rb

```diff
@@ -141,15 +141,15 @@ def perform_request(gcse_requirements)
   end
 
   context "when not_set is provided on a secondary course" do
-    let(:secondary_subject) { build(:ucas_subject, :secondary) }
+    let(:secondary_subject) { build(:subject, type: 'SecondarySubject') }
```

Comment:
can we have that as a trait or use one of the existing please

Reply:
Done 👍